doc/whats_new/v0.21.rst

```diff
@@ -167,7 +171,7 @@ Support for Python 3.4 and below has been officially dropped.
 
 - |Fix| Fixed a bug in :class:`linear_model.LassoLarsIC`, where user input
    ``copy_X=False`` at instance creation would be overridden by default
-   parameter value ``copy_X=True`` in ``fit``. 
+   parameter value ``copy_X=True`` in ``fit``.
```

Comment:
Please avoid fixing pep8 issues not related to your PR. It makes reviews harder to follow

Reply:
Ok. I will avoid it for future PRs, but I guess I should keep this one.

Reply:
we usually ask contributors to revert it :/